scripts/tests/test_tractogram_commit.py

```diff
@@ -20,27 +27,31 @@ def test_help_option(script_runner):
 
 
 def test_execution_commit_amico(script_runner, monkeypatch):
+    tmp_dir = tempfile.TemporaryDirectory()
     monkeypatch.chdir(os.path.expanduser(tmp_dir.name))
-    in_tracking = os.path.join(SCILPY_HOME, 'commit_amico',
-                               'tracking.trk')
-    in_dwi = os.path.join(SCILPY_HOME, 'commit_amico',
-                          'dwi.nii.gz')
-    in_bval = os.path.join(SCILPY_HOME, 'commit_amico',
-                           'dwi.bval')
-    in_bvec = os.path.join(SCILPY_HOME, 'commit_amico',
-                           'dwi.bvec')
-    in_mask = os.path.join(SCILPY_HOME, 'commit_amico',
-                           'mask.nii.gz')
-    in_peaks = os.path.join(SCILPY_HOME, 'commit_amico',
-                            'peaks.nii.gz')
-    ret = script_runner.run('scil_tractogram_commit.py', in_tracking, in_dwi,
-                            in_bval, in_bvec, 'results_bzs/',
-                            '--b_thr', '30', '--nbr_dir', '500',
-                            '--nbr_iter', '500', '--in_peaks', in_peaks,
-                            '--in_tracking_mask', in_mask,
-                            '--para_diff', '1.7E-3',
-                            '--perp_diff',
-                            '1.19E-3', '0.85E-3', '0.51E-3', '0.17E-3',
-                            '--iso_diff', '1.7E-3', '3.0E-3',
-                            '--processes', '1')
+
+    ret = script_runner.run(
+        'scil_tractogram_commit.py', in_tracking, in_dwi, in_bval, in_bvec,
+        'results_bzs/', '--tol', '30', '--nbr_dir', '500',
+        '--nbr_iter', '500', '--in_peaks', in_peaks,
+        '--in_tracking_mask', in_mask,
+        '--perp_diff', '1.19E-3', '0.85E-3', '0.51E-3', '0.17E-3',
+        '--iso_diff', '1.7E-3', '3.0E-3',
+        '--processes', '1', '--keep_whole_tractogram')
     assert ret.success
+
+
+def test_execution_commit2(script_runner, monkeypatch):
+    tmp_dir = tempfile.TemporaryDirectory()
+    monkeypatch.chdir(os.path.expanduser(tmp_dir.name))
+
+    # TODO Modify to use with hdf5. Currently impossible.
```

Comment:
How come ?

Reply:
I modified to `# TODO Add a HDF5 in our test data that could be used here.` So there only remains the question of wheter we should add new test data right now. 

Considering that there will be changes in the test data management soon, I would leave the todos and manage this in another PR.